db/migrate/20181001131632_add_conversion_host_id_to_miq_request_tasks.rb

```diff
@@ -0,0 +1,46 @@
+class AddConversionHostIdToMiqRequestTasks < ActiveRecord::Migration[5.0]
+  class MiqRequestTask < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options, Hash
+    belongs_to :conversion_host, :class_name => "AddConversionHostIdToMiqRequestTasks::ConversionHost"
+  end
+
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    belongs_to :resource, :polymorphic => true
+  end
+
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :miq_request_tasks, :conversion_host_id, :bigint
+
+    MiqRequestTask.where(:type => 'ServiceTemplateTransformationPlanTask').each do |task|
+      host_id = task.options.delete(:transformation_host_id)
+      next unless host_id
+      host = Host.find_by(:id => host_id)
+      if host.present?
+        task.conversion_host = ConversionHost.find_or_create_by!(:resource => host) do |ch|
```

Comment:
`ConversionHost.find_or_initialize_by(:resource_type => "Host", :resource_id => host.id).update_attributes!(.....)`

Reply:
I don't see why we should use `find_or_initialize_by` + `update_attributes` over what's here, but yeah it does look like this will break because the host class will be incorrect.